src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -16,44 +16,71 @@ UI Options does three things:
 
 ## Scenario
 
 You're putting together a website that you know will have a diverse audience. You'd like to allow your visitors to
 customize the presentation of the site to their individual needs, by enlarging the text or increasing the visual
-contrast, for example. This tutorial will show you how to add the Infusion [UI
-Options](http://build.fluidproject.org/infusion/demos/uiOptions/) component to your site.
+contrast, for example. This tutorial will show you how to add the Infusion UI Options component to your site.
 
 These are the basic steps to add UI Options to your website:
 
-1. [Download and install the Infusion library](#download-and-install-the-ui-options-library)
+1. [Download and install the UI Options library](#download-and-install-the-ui-options-library)
 2. [Prepare your page](#prepare-your-page)
     * [The Sliding Panel](#the-sliding-panel)
     * [The Table of Contents](#the-table-of-contents)
 3. [Add dependencies to the page](#add-dependencies-to-the-page)
 4. [Add the UI Options component](#add-the-ui-options-component)
 
-The rest of this tutorial will explain each of these steps in detail.
+The rest of this tutorial will explain each of these steps in detail. This tutorial also assumes that you can work with HTML, Javascript and CSS.
 
 ## Download and install the UI Options library
 
+Get Infusion using one of these methods:
+* [Zip Package](https://github.com/fluid-project/infusion/releases/download/v2.0.0/infusion-uiOptions-2.0.0-source.zip)
+* [Github source](https://github.com/fluid-project/infusion)
+* npm via `npm i infusion`
+
+### Zip Package Method
+This method will download and install the last release of Infusion (2.0.0).
+
 1. [Download the UI Options library](https://github.com/fluid-project/infusion/releases/download/v2.0.0/infusion-uiOptions-2.0.0-source.zip)
   (there is also a [minified version UI Options](https://github.com/fluid-project/infusion/releases/download/v2.0.0/infusion-uiOptions-2.0.0-minified.zip)
    for deployment purposes).
 2. Unzip the contents of the downloaded ZIP file to a location within your project. This will result in a new directory
    `infusion`.<div class="infusion-docs-note"><strong>Note:</strong> In this guide we will use the directory
    `my-project/lib/`.</div>
-3. Your `infusion` folder will include a single file containing all of the JavaScript you need
+3. Your `infusion` directory will include a single file containing all of the JavaScript you need
    (`infusion-uiOptions.js`), HTML templates, CSS files, and other components to get UI Options to work. You will later
    link to these files in your HTML files.
 4. Now that `infusion` is in your project directory, you can delete the `infusion-uiOptions-2.0.0.zip` (or similar name)
    from your download directory.
 
+### Github Source Method
+This method will download and install the most recent development version of Infusion.
```

Comment:
I guess there is something slightly confusing with our current Infusion process we have 

- "official/stable" releases that have undergone our full test process, been tagged, and versioned. 
- dev releases which are published to NPM and are more or less points in time that someone feels marks a "milestone" of code. These may or may not all work. Sometimes there is still a bug and a new dev release is made.
- Master, this is just the latest committed code. We try not to break master though, so it is usually relatively stable in terms of tests passing and obvious bugs, but as mentioned above things do creep in from time to time. 

When you say most recent development version here, it's not clear to me that the latest dev release and master may not point at the same code.

Reply:
Yeah. This is getting confusing because depending which of the 3 methods you follow, you can possibly end up with 3 different versions of Infusion.

While this may not be an issue for a single user, people working in teams may end up with different versions depending on which approach they take.

Some of this can be addressed by documenting this, but it can still be confusing.